src/SeaIceDeformation/utils_compute_deformations.py

```diff
@@ -129,6 +129,7 @@ def compute_deformations(config=None):
             sLon    = raw_data['sLon']    # Starting longitudes
             eLat    = raw_data['eLat']    # Ending latitudes
             eLon    = raw_data['eLon']    # Ending longitudes
+            sat     = (0)*('rcm' in raw_path)+(1)*('s1' in raw_path)+(2)*('rcm_new' in raw_path)
```

Comment:
```suggestion
            sat     = (0)*('rcm' in raw_path)+(1)*('s1' in raw_path)+(2)*('rcm_new' in raw_path) # If we have 'rcm_new' in the path, it will return both 0+2 = 2, so it works for now, but we may want change it eventually.
```